docker/fetch_normalization_data.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python
+
+"""
+Fetches normalization data.
+
+Usage:
+
+    python docker/fetch_normalization_data.py
+
+"""
+
+import argparse
+import datetime
+import os
+import sys
+
+import psycopg2
+import requests
+
+
+# NOTE(willkg): These platforms come from the -prod data.
+PLATFORMS = ['Mac OS X', 'Linux', 'Windows', 'Unknown']
+
+
+def fetch_data_from_api(endpoint, product, version, platform, start_date, end_date):
+    """Fetches data from API"""
+    req = requests.get(endpoint, params={
+        'end_date': end_date,
+        'start_date': start_date,
+        'product': product,
+        'platforms': platform,
+        'versions': version,
+    })
+    if req.status_code != 200:
+        print('Error retrieving ADI: %s' % req.content)
+        return []
+
+    data = req.json()
+    return data['hits']
+
+
+def fetch_versions(conn, product_name):
+    """Fetch the version strings for a product from product_versions table"""
+
+    cursor = conn.cursor()
+    cursor.execute("""
+        SELECT version_string
+        FROM product_versions
+        WHERE
+            product_name=%s
+    """, (product_name,))
+    # NOTE(willkg): We don't want versions that end with "b" because they're weird.
+    versions = set([record[0] for record in cursor if not record[0].endswith('b')])
```

Comment:
>  If you do and `/api/ADI` query for `?versions=55.0b&versions=55.0b3` you get the same output as if you'd done `?versions=55.0b3`.

It's not... The latter only gets you ADI for that specific version and the former gets you ADI for all the beta versions including that version:

55.0b and 55.0b3:

https://crash-stats.mozilla.com/api/ADI/?end_date=2017-08-10&platforms=Mac%20OS%20X&product=Firefox&start_date=2017-08-03&versions=55.0b&versions=55.0b3

just 55.0b3:

https://crash-stats.mozilla.com/api/ADI/?end_date=2017-08-10&platforms=Mac%20OS%20X&product=Firefox&start_date=2017-08-03&versions=55.0b3

Thank you for explaining what was going on! I think I'm going to leave it in because I don't want to deal with a prefix match.